tests/roles/stop_openstack_services/templates/systemctl_disable.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Disable and mask the service on the Computes.
+service=$1
+
+cat << EOF > ~/disable_service.yaml
```

Comment:
This seems to be getting quite far from what's written in the docs... If there are OSPDO specifics that make us go through extra steps / indirection, then that's fine but they should probably be added to the docs too.

Can we just use a SSH command to stop the services like we're doing for the other environment? We don't assume that the end user will have Ansible at their disposal.